test/UserOp.ts

```diff
@@ -83,7 +77,10 @@ export const DefaultsForUserOp: UserOperation = {
   preVerificationGas: 21000, // should also cover calldata cost.
   maxFeePerGas: 0,
   maxPriorityFeePerGas: 1e9,
-  paymasterAndData: '0x',
+  paymaster: '0x',
+  paymasterData: '0x',
+  paymasterVerificationGasLimit: 3e5,
+  paymasterPostOpGasLimit: 0,
```

Comment:
why default is zero? yes, it does raise the paymaster required balance a little, but simplify using it.
`fillUserOp` can ignore (set to zero) the those pm gas limits if paymaster address is zero

Reply:
Why not zero?

Reply:
it is a helper for tests, and now it clutters all tests with the need to add it.

Reply:
Fixed, forgot to remove that